### PR TITLE
Doc issue patch

### PR DESCRIPTION
**Pull request by tyg**
Darn, should have gotten this out before all the merging of today's quick new docs creation. Oh well, I should be able to work out the merge. This is just a bunch of small bits to either fix some typos or close out various Documentation issues, mostly entered by David.

### CODE REVIEW

File: main/getting-started/ertp-introduction.md [hunk 1]
```diff
@@ -54,7 +55,8 @@ ERTP guarantees:
 Other key ERTP objects are:
 
 - **Mints**: Issue new digital assets as a new Payment. Mints only
-issue one type of asset (quatloos, simoleons, moola, etc.). We refer to
+issue one type of asset (quatloos, simoleons, moola, etc. Note that these
+are all imaginary currencies used as examples.). We refer to
```
Comment (katelynsills) [inline main/getting-started/ertp-introduction.md:59]:
```suggestion
are all imaginary currencies used as examples.) We refer to
```

File: main/getting-started/intro-zoe.md [hunk 2]
```diff
@@ -189,7 +194,7 @@ Let’s look at a more complicated example: an auction, where many users might m
...
 
 
-It’s even possible to get both what you wanted in an offer, and at least a partial refund of what you offered. Consider an auction where you make a bid offer where you’re willing to pay up to 10 quatloos, but if you can win with a lower bid, that’s your offer. You win the auction, but your winning bid amount is only 8 quatloos. The payout would resolve to both the item up for auction you get as the winning bidder and the refund of the 2 extra quatloos you escrowed in the offer. 
+It’s even possible to get both what you wanted in an offer, and at least a partial refund of what you offered. Consider an auction where you make a bid offer where you’re willing to pay up to 10 quatloos (an imaginatry currency), but if you can win with a lower bid, that’s your offer. You win the auction, but your winning bid amount is only 8 quatloos. The payout would resolve to both the item up for auction you get as the winning bidder and the refund of the 2 extra quatloos you escrowed in the offer.
```
Comment (katelynsills) [inline main/getting-started/intro-zoe.md:197]:
```suggestion
It’s even possible to get both what you wanted in an offer, and at least a partial refund of what you offered. Consider an auction where you make a bid offer where you’re willing to pay up to 10 quatloos (an imaginary currency), but if you can win with a lower bid, that’s your offer. You win the auction, but your winning bid amount is only 8 quatloos. The payout would resolve to both the item up for auction you get as the winning bidder and the refund of the 2 extra quatloos you escrowed in the offer. 
```

File: main/glossary/README.md [hunk 3]
```diff
@@ -26,6 +26,23 @@ Purses and Payments are AssetHolders.
 ## Brand
 Identifies the kind of issuer.
 
+## Comparable
+
+A *passable* is something that can be mashalled. A *comparable* is a
```
Comment (katelynsills) [inline main/glossary/README.md:31]:
```suggestion
A *passable* is something that can be marshalled. A *comparable* is a
```

File: main/glossary/README.md [hunk 4]
```diff
@@ -56,16 +73,44 @@ Arithmetic on extents. MathHelpers are used by AmountMath to do their extent ari
...
+- Construction: Constuctors can endow their constructed objects with  references, including inherited references. 
+- Introduction: 
+  - A has references to B and C. 
+  - B and C  do *not* have references to each other
+  - A sends B a reference to C. 
+    - B now has a reference to C and can communicate with C. 
+
+If references can only be obtained by creation, construction, or introduction, you may have a safe system. If they can be obtained in any other way, your system is unsafe.
+
+For more information, see [Douglas Crockford on Object Capabilities](https://frontendmasters.com/courses/good-parts-javascript-web/object-capabilities/).
```
Comment (katelynsills) [inline main/glossary/README.md:99]: Do users have to pay for this?

File: main/glossary/README.md [hunk 5]
```diff
@@ -26,6 +26,23 @@ Purses and Payments are AssetHolders.
...
 
+## Comparable
+
+A *passable* is something that can be mashalled. A *comparable* is a
+passable whose leaves contain no promises. Two comparables can be
+synchronously compared for structural equivalence.
+
+A comparable is a JavaScript object containing no promises, and can
+thus be locally compared for equality with another object. If either object
+contains promises, equality is indeterminable. If both are fullfilled down
```
Comment (Chris-Hibbert) [inline main/glossary/README.md:37]: equality can't be decided.

File: main/glossary/README.md [hunk 6]
```diff
@@ -56,16 +73,44 @@ Arithmetic on extents. MathHelpers are used by AmountMath to do their extent ari
...
+Objects have state, behavior, and references. Lets say Object A has references to Objects B and C, while B and C do not have references to each other. Thus, A can communicate with B and C, and B and C cannot commuicate with each other.
+There is an effective zero-cost firewall between B and C.
+
+An *object capability system* constrains how references are obtained. You can't get one just by knowing the name of a global variable or a public class. You can get a reference in only three ways. 
+- Creation: Functions that create objects get a reference to them.
+- Construction: Constuctors can endow their constructed objects with  references, including inherited references. 
+- Introduction: 
+  - A has references to B and C. 
+  - B and C  do *not* have references to each other
+  - A sends B a reference to C.
```
Comment (Chris-Hibbert) [inline main/glossary/README.md:94]: How about "A sends a message to B containing a reference to C"?

**Pull request merged**